### DISCUSSION TITLE
Fix the 301 search engine notice that hurts my traffic

### DISCUSSION THREAD

**Discussion by qqiumax** [community/community#23139] [Other Feature Feedback, Questions, & Ideas]
<h2>
<a name="when-i-searched-my-github-page-on-bing-found-this-ugly-message-1" class="anchor" href="#when-i-searched-my-github-page-on-bing-found-this-ugly-message-1"></a>When I searched my Github page on Bing, found this ugly message:</h2>
<p><img src="https://user-images.githubusercontent.com/102963434/181075092-eab98dd2-2502-43a1-b4fe-9a6b847caf05.jpeg" alt="Web capture_19-5-2022_145251_www.bing.com" data-base62-sha1="aXl5F9SNET9MiiG1TUpgeSmTZtY" width="475" height="117"></p>
<h2>
<a name="it-is-such-bad-that-it-cant-show-previews-and-my-title-which-hurts-my-traffic-how-can-i-fix-it-2" class="anchor" href="#it-is-such-bad-that-it-cant-show-previews-and-my-title-which-hurts-my-traffic-how-can-i-fix-it-2"></a>It is such bad that it can’t show previews and my title, which hurts my traffic, how can I fix it?</h2>

**Comment by mpboom** [comment, marked as answer]
<p>Bing just decides to display the body of the 301 redirect. Quite stupid, but there isn’t much that either you or GitHub can do to prevent it. Note the following things though:</p>
<ul>
<li>
<code>/Blog</code> is a 404 page, which always works wonky in search engines</li>
<li>Sometimes a bit of patience goes a long way</li>
<li>Keep up good SEO practice: fill your site with semi-regularly updated original content</li>
<li>Nobody really uses Bing</li>
</ul>

**Comment by jfullstackdev** [comment]
<p>the user changed his / her username but now it’s already 404 too</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/b/8b77556cb53f02b3647f013aec6f7b800f2b0d65.jpeg" data-download-href="/uploads/short-url/jTM2soiClPBDja37JZR7ncLGNWR.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181075097-0a9060f4-b882-4763-aab9-136d9ef3fbb0.jpeg" alt="image" data-base62-sha1="jTM2soiClPBDja37JZR7ncLGNWR" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8b77556cb53f02b3647f013aec6f7b800f2b0d65_2_690x373.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8b77556cb53f02b3647f013aec6f7b800f2b0d65_2_1035x559.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8b77556cb53f02b3647f013aec6f7b800f2b0d65_2_1380x746.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8b77556cb53f02b3647f013aec6f7b800f2b0d65_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 222 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by qqiumax** [comment]
<p>Truth :)))))))))))))))))))))))))))))))))))))))))))))))))))))))))</p>
